### ISSUE TITLE
Learning rate not decreasing after warmup

### ISSUE THREAD

**Issue by curiousily** [facebookresearch/detectron2#497]
Hello,

I am fine-tuning a model on a custom dataset and the learning rate rate never seems to go down after the warmup period. Here are the relevant configs:

```py
cfg = get_cfg()
cfg.merge_from_file("./detectron2_repo/configs/COCO-InstanceSegmentation/mask_rcnn_X_101_32x8d_FPN_3x.yaml")
cfg.DATASETS.TRAIN = ("custom_train",)
cfg.DATASETS.TEST = ("custom_val",)
cfg.DATALOADER.NUM_WORKERS = 2
cfg.MODEL.WEIGHTS = "detectron2://COCO-InstanceSegmentation/mask_rcnn_X_101_32x8d_FPN_3x/139653917/model_final_2d9806.pkl"
cfg.SOLVER.IMS_PER_BATCH = 2
cfg.SOLVER.BASE_LR = 0.00001
cfg.SOLVER.WARMUP_ITERS = 100
cfg.SOLVER.MAX_ITER = 2000
cfg.MODEL.ROI_HEADS.BATCH_SIZE_PER_IMAGE = 128
cfg.MODEL.ROI_HEADS.NUM_CLASSES = 1
```

I am training on a single GPU Tesla P100. Initially, the learning rate start at around 0 and goes up to the BASE_LR (expected). However, the learning rate never goes down after 100 iters. Is there any place where the meaning of `MAX_ITER`, `WARMUP_ITERS` and `STEPS` are documented?

Thanks for making and opening Detectron2. It is truly amazing work!

**Comment by ppwwyyxx** [comment]
They are documented in https://detectron2.readthedocs.io/modules/config.html#config-references
